src/frontend/screens/MetaMaskPortfolio/index.tsx

```diff
@@ -13,7 +13,7 @@ const MetaMaskPortfolio = function () {
         <div className={MetaMaskPortfolioStyles.loader}></div>
         <webview
           className={MetaMaskPortfolioStyles.homeWebview}
-          src={`https://portfolio.metamask.io/${page}`}
+          src={`https://portfolio.metamask.io/${page}?utm_source=hyperplay`}
```

Comment:
Maybe would be good to use variables and a URL constructor to handle this on a variable instead of directly here.

Reply:
sgtm. I moved it to the constants module as a function. I'd prefer to just keep it simple for now but if the url gets more complicated, we could refactor to URL object